src/desktop/apps/consign/client/actions.js

```diff
@@ -280,7 +285,7 @@ export function fetchArtistSuggestions(value) {
       const res = await request
         .get(`${sd.API_URL}/api/v1/match/artists`)
         .query({ visible_to_public: "true", term: value })
-        .set("X-ACCESS-TOKEN", user.accessToken)
+        .set("X-XAPP-TOKEN", sd.ARTSY_XAPP_TOKEN)
```

Comment:
Yup exactly. In order to query Gravity's API we need _some_ kind of token. Before we were using the user because it was around, but now we're leaning on the access token.